packages/moonstone/Scroller/Scroller.js

```diff
@@ -262,16 +232,14 @@ class ScrollerBase extends Component {
 			return;
 		}
 
-		const {
-			top: itemTop,
-			left: itemLeft,
-			height: itemHeight,
-			width: itemWidth
-		} = this.getFocusedItemBounds(item);
-
 		if (this.uiRef.isVertical()) {
-			this.uiRef.scrollPos.top = this.calculateScrollTop(item, itemTop, itemHeight, scrollInfo, scrollPosition);
+			this.uiRef.scrollPos.top = this.calculateScrollTop(item, scrollInfo, scrollPosition);
```

Comment:
Do we need `scrollInfo`, `scrollPosition` parameters because there is a only `item` parameter in `calculateScrollTop`?

Reply:
I'm not sure. I expected to need it but they aren't right now. If the functionality checks out as is, we should remove passing them.